ansible/roles/vmware-template/tasks/deploy.yml

```diff
@@ -45,7 +45,7 @@
 
     datacenter: "{{ datacenter_name }}"
 
-    uuid: "{{ new_vm.instance.hw_product_uuid }}"
+    id: "{{ new_vm.instance.hw_product_uuid }}"
```

Comment:
Are you certain that `vmware_guest_facts` accepts `id` as a parameter? According to https://github.com/ansible/ansible/blob/v2.4.0.0-1/lib/ansible/modules/cloud/vmware/vmware_guest_facts.py#L132 it accepts `uuid`, not `id`.

Reply:
You're right. I haven't intended to make this change.